### DISCUSSION TITLE
Utf8's/emotes

### DISCUSSION THREAD

**Discussion by majidaniel** [Kode/Kha#1493] [Q&A] [answered]
Hello,
I am trying to use emotes/symbols from other language fonts using Kha.drawString ala:
`var str = "test ☺️🙂😊😀😁 ♉ 🪦";
        g2.font = Assets.fonts.NotoEmoji;
        g2.drawString(str,0,0);
        `

I have repeated the above using drawCharacters and the int values inside of the font, and had the same results.  They do not display (NotoEmoji & Arial shown in image):
![image](https://github.com/Kode/Kha/assets/4972528/53ef6f7f-f620-4382-9601-d65d677cb803)

Is unicode not supported? Or can someone please point me in a direction to look into?
Thanks!

**Comment by RobDangerous** [comment, marked as answer]
Sorry for the late answer, missed this one :(
Unicode is supported but you have to use a font that includes what you want and you have to set the unicode range you want to use via [fontGlyphs](https://kha.tech/api/kha/graphics2/Graphics.html#fontGlyphs).
